src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed') || false;
 
             /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
             richTextEditorBuilder.initEditor(`#${id}`, {
                 inline: true,
+                readonly: isSessionClosed,
```

Comment:
The main reason I don't like this is that even when directly assigning `readonly` to `isSessionClosed`, the reader would still have to pause to think why the two are equivalent. Not sure if you agree with me, but I'd argue that most readers still go through this thought process:
```javascript
readonly: isSessionClosed ? true : false,
```
If `readonly` is instead assigned to `!isSessionOpen`, the thought process would be more like:
```javascript
readonly: isSessionOpen ? false : true,
```
In fact, the above might be the most readable.